Cargo.toml

```diff
@@ -22,7 +22,7 @@ all-features = true
 miniquad = "0.3.0-alpha.25"
 quad-rand = "0.1"
 glam = {version = "0.10", features = ["scalar-math"] }
-image = { version = "0.22", default-features = false, features = ["png_codec", "tga"] }
+image = { version = "0.23.12", default-features = false, features = ["png", "tga"] }
```

Comment:
Maybe use more relaxed "0.23" here instead?

Reply:
I am not sure. The advisory specifically mentions Patched Versions >=0.23.12. Afaik cargo selects the newest version possible when resolving deps *but* if somebody already has a version like 0.23.0 in cargo.lock, it won't update for him. Putting 0.23.12 in macroquad forces their project to update too. At least i think. OTOH the situation itself is unlikely here - somebody would have to depened on image 0.22 through macroquad and separately on 0.23 directly or through some other dep. I am wondering about this mostly for future reference not because it's likely to happen here.

Ideally, people would notice things like this because they'd have cargo audit in CI and deps.rs in readme but most projects don't. :P

I never wrote a rust lib so idk what the guidelines are - are there any? Theoretically, anything between 0.23.0 and 0.23.12 should be only compatible changes so requiring 0.23.12 shouldn't break anybody's build.

Reply:
If we depend on 0.23.12 than someone with image 0.23.13 in their deps would have to versions of image compiled and added to the build artifacts. 
So it is really a good idea to allow cargo to select the best compatible image version :)

Reply:
I don't think so. According to https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html and https://doc.rust-lang.org/cargo/reference/resolver.html , `0.23.12` in Cargo.toml means `>=0.23.12, <0.24` so it should allow any newer version which is compatible (0.23.x where x >= 12) but not older ones with the UB.

Reply:
ok, lets keep 0.23.12 :)